### PR TITLE
General Corral and Feild

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/skirmish/structures/default_corral.xml [hunk 1]
```diff
@@ -5,6 +5,9 @@
   </Footprint>
   <Identity>
     <Civ>skirm</Civ>
+    <History>
+         In ancient times meat was eaten much less frequently than today. It was far better to keep a sheep, for her and milk, or a hen for her eggs. Most often the lower class would eat the meat from temple sacrifices. What wasn’t burnt on the altar was often cooked and shared between the priests and the people.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/skirmish/structures/default_corral.xml:9]:
```suggestion
         In ancient times meat was eaten much less frequently than today. It was far better to keep a sheep, for her milk, or a hen for her eggs. Most often the lower class would eat the meat from temple sacrifices. What wasn’t burnt on the altar was often cooked and shared between the priests and the people.
```

File: 0-ad-history-mod-package/simulation/templates/skirmish/structures/default_field.xml [hunk 2]
```diff
@@ -2,6 +2,9 @@
 <Entity parent="template_structure_resource_field">
   <Identity>
     <Civ>skirm</Civ>
+    <History>
+        Grains such as wheat and barely were staples for any ancient society. They were cheap, easy to grow crops that were a staple in the diets of all but the richest individuals.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/skirmish/structures/default_field.xml:6]:
```suggestion
        Grains such as wheat and barley were staples for any ancient society. They were cheap, easy-to-grow crops that were a staple in the diets of all but the richest individuals.
```

**Review by indoptogopt — CHANGES_REQUESTED** [review]

**Comment by TheShadowOfHassen** [comment]
Is this OK to merge.

**Comment by indoptogopt** [comment]
Yes

**Pull request merged**